pycbc/injcutter.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (C) 2013 Ian Harry
```

Comment:
So to understand. Suggestion is to create an injection package and then move injcutter.py and inject.py into that package?

This will require editing all codes that use inject.py, but okay ... Just want to be sure of the suggestion first.